plugins/system/deborphan

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
Not really, as I'm using some bashism to do some quick pattern replacement later on.